packs/feats/strix-vengeance.json

```
@@ -11,7 +11,7 @@
         },
         "category": "ancestry",
         "description": {
-            "value": "<p><strong>Frequency</strong> once per 10 minutes</p>\n<p><strong>Trigger</strong> You, or a strix ally you can see, are damaged by an enemy's critical hit.</p>\n<hr />\n<p>You dedicate yourself to destroying those who harm your kin. Until the end of your next turn, you deal an additional [[/r 1d6]]{+1d6 damage} on Strikes against the triggering enemy. The bonus increases to [[/r 2d6]]{+2d6 damage} if you use a Striking weapon or unarmed attack and [[/r 3d6]]{+3d6 damage} if you use a Striking (Major) weapon or unarmed attack.</p>"
+            "value": "<p><strong>Frequency</strong> once per 10 minutes</p>\n<p><strong>Trigger</strong> You, or a strix ally you can see, are damaged by an enemy's critical hit.</p><hr /><p>You dedicate yourself to destroying those who harm your kin. Until the end of your next turn, you deal an additional @Damage[1d6]{+1d6 damage} on Strikes against the triggering enemy. The bonus increases to @Damage[2d6]{+2d6 damage} if you use a Striking weapon or unarmed attack and @Damage[3d6]{+3d6 damage} if you use a Striking (Major) weapon or unarmed attack.</p>"
```

```suggestion
            "value": "<p><strong>Frequency</strong> once per 10 minutes</p>\n<p><strong>Trigger</strong> You, or a strix ally you can see, are damaged by an enemy's critical hit.</p><hr /><p>You dedicate yourself to destroying those who harm your kin. Until the end of your next turn, you deal an additional +1d6 damage on Strikes against the triggering enemy. The bonus increases to +2d6 damage if you use a Striking weapon or unarmed attack and +3d6 damage if you use a Striking (Major) weapon or unarmed attack.</p>"
```

If this isn't automated properly we should automate it